Doc/tutorial/errors.rst

```diff
@@ -400,7 +400,7 @@ If a :keyword:`finally` clause is present, the :keyword:`finally` clause will ex
 
 For example::
 
-   >>> def bool_return(): -> bool:
+   >>> def bool_return() -> bool:
```

Comment:
I do not think this function should use a return annotation. No other code or examples (except related to the typing module) use annotations. I think it was intended as
```
def bool_return():  #  -> bool
```
Or maybe ` -> bool:` should be removed at all.

Reply:
agreed, happy to make that adjustment.

Reply:
Seconding the

> Or maybe -> bool: should be removed at all.

suggestion.